### PR TITLE
API for saving and loading settings

### PR DESCRIPTION
**Pull request by lwesterhof**
Uses the sudo policies to allow setting a AVU as a normal rodsuser.

### CODE REVIEW

**Comment by HarmdR** [comment]
Ziet er prima uit 👍

File: settings.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    :param settings: Dict with settings to be saved
+
+    :returns: API status
+    """
+    for a, v in settings.items():
+        if a in USER_SETTINGS and v in USER_SETTINGS[a]["values"]:
+            try:
+                ctx.uuUserModify(user.full_name(ctx), "{}{}".format(SETTINGS_KEY, a), str(v), '', '')
+            except Exception as e:
+                return api.Error('internal', 'Saving settings failed. Please try again')
```
Comment (reavdhoef) [inline settings.py]: Does this error message end up at the user? If so, I would suggest to change the error message to something along the lines of "Saving settings failed. Please try again shortly. If the problem persists after a few tries, please contact Yoda FB."

**Review by reavdhoef — APPROVED** [review]
Small remark on error message, but looks good.

**Review by HarmdR — APPROVED** [review]

File: settings.py (hunk 1 above)
Comment (lwesterhof) [reply settings.py]: Thanks, done.

**Pull request merged**